lib/bitgo_client/v2.rb

```diff
@@ -60,8 +64,18 @@ def transaction(wallet_id, transaction_id, coin_code: :tbtc, logger: nil)
       client.request("#{base_path}/#{coin_code}/wallet/#{wallet_id}/tx/#{transaction_id}", logger: logger)
     end
 
-    def fee(coin_code: :tbtc, logger: nil)
-      client.request("#{base_path}/#{coin_code}/tx/fee", logger: logger)
+    def transfers(wallet_id, coin_code: :tbtc, logger: nil, limit: 25, prev_id: nil, all_tokens: nil)
+      query_string = build_query_string(
+        limit:     limit,
+        prevId:    prev_id,
+        allTokens: all_tokens
```

Comment:
why prevId and allToken are in camel case ? or they should be "prev_id" and "all_token" ?

Reply:
Those are query string params for Bitgo requests

Reply:
👍